src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, String color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        setColor(color);
+        obtainTheArea();
+    }
+
+    public void obtainTheArea() {
+        setArea(firstSide * secondSide);
+    }
+
+    @Override
+    public String draw() {
```

Comment:
The `draw` method should not be responsible for calculating the area. The area should be calculated and set before this method is called.